app/src/main/java/com/col/eventradar/ui/adapters/EventCommentRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package com.col.eventradar.ui.adapters
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.col.eventradar.databinding.EventCommentBinding
+
+data class Comment(val content: String, val username: String, val time: String)
```

Comment:
```suggestion
data class Comment(val content: String, val username: String, val time: LocalDateTime)
```
or

```suggestion
import com.google.firebase.Timestamp

data class Comment(val content: String, val username: String, val time: Timestamp? = null)
```
I think will be better suited for project then just a regular `string`

Reply:
yeah but its just mock data for now. so itss easier to work with strings. once we use actual data it will be an actual date. ok?

Reply:
but you want to make it ready to get real data. and work with that and make formatting for real dates that are coming to it. I think it is important that not only the visual of the fragment will be ready but the base types of data that it works with as well. in any case we will that in future

Reply:
👍